src/Avalonia.Controls/SplitView.cs

```diff
@@ -394,24 +397,24 @@ private bool ShouldClosePane()
             return (DisplayMode == SplitViewDisplayMode.CompactOverlay || DisplayMode == SplitViewDisplayMode.Overlay);
         }
 
-        protected virtual void OnPaneOpening(SplitView sender, EventArgs args)
+        protected virtual void OnPaneOpening(EventArgs args)
```

Comment:
Removed sender parameter from these protected virtual methods. It is not usually provided and is redundant (OnPropertyChanged itself is an example).